content/javascript/concepts/arrays/terms/filter/filter.md

```diff
@@ -115,3 +115,16 @@ The output would be:
   team: "Red"
 }]
 ```
+
+## Codebyte Example
+
+Return an `array` of all values in ages[] that are 18 or over:
+
+```codebyte/javascript
+const ages = [32, 33, 16, 40];
+const result = ages.filter(checkAdult);
+
+function checkAdult(age) {
+  return age >= 18;
+}
+```
```

Comment:
Looks this is missing the output call -

console.log(result)

Reply:
right thank you!